demo/components/Translate.tsx

```diff
@@ -0,0 +1,14 @@
+const translate = () => {
```

Comment:
Can you use the function component and make it PascalCase, just a convention :)
```suggestion
function translate(){
```

Reply:
Sure. Fixed by https://github.com/nottyo/jest-preview/commit/9c391798b339f7fb13dab952b339102a0d8bfdcf